File: src/main/java/ru/ioleksiv/telegram/bot/api/model/annotation/validator/impl/EncryptedPassportElementFilterValidator.java
package ru.ioleksiv.telegram.bot.api.model.annotation.validator.impl;

import ru.ioleksiv.telegram.bot.api.model.annotation.validator.CustomValidator;
import ru.ioleksiv.telegram.bot.api.model.annotation.validator.FilterValidator;
import ru.ioleksiv.telegram.bot.api.model.objects.passport.EncryptedPassportElement;

public class EncryptedPassportElementFilterValidator extends FilterValidator<EncryptedPassportElement> {

    public EncryptedPassportElementFilterValidator(CustomValidator<EncryptedPassportElement> validator) {
        super(EncryptedPassportElement.class, validator);
    }
}

File: core/src/main/kotlin/io/github/oleksivio/tl/kbot/core/model/annotation/validator/impl/EncryptedPassportElementFilterValidator.kt
package io.github.oleksivio.tl.kbot.core.model.annotation.validator.impl

import io.github.oleksivio.tl.kbot.core.controller.handler.check.Validator
import io.github.oleksivio.tl.kbot.core.model.annotation.validator.FilterValidator
import io.github.oleksivio.tl.kbot.server.api.objects.passport.EncryptedPassportElement

class EncryptedPassportElementFilterValidator(validator: Validator<EncryptedPassportElement>) :
    FilterValidator<EncryptedPassportElement>(
        EncryptedPassportElement::class, validator
    )
